src/rust/ide/view/graph-editor/src/component/node/port.rs

```diff
@@ -149,6 +150,7 @@ impl Manager {
             hover_source        <- source::<Option<span_tree::Crumbs>>();
             start_edit_mode     <- source();
             stop_edit_mode      <- source();
+            active              <- label.active.sampler();
```

Comment:
what does it mean that port is "active"?

Reply:
It means that the label is active so it can be edited. I'll rename that.

Reply:
rename to `editing`